setup.cfg

```diff
@@ -25,17 +86,6 @@ addopts = --doctest-rst
 [ah_bootstrap]
 auto_use = True
```

Comment:
Can this `[ah_bootstrap]` section be removed?

Reply:
Yes, good catch!